packages/renderer/src/stores/kubernetes-contexts-state.ts

```diff
@@ -246,3 +247,15 @@ export const kubernetesCurrentContextRoutesFiltered = derived(
   [routeSearchPattern, kubernetesCurrentContextRoutes],
   ([$searchPattern, $routes]) => $routes.filter(route => findMatchInLeaves(route, $searchPattern.toLowerCase())),
 );
+
+// Port Forwarding
+
+export const kubernetesCurrentContextPortForwards = readable<UserForwardConfig[]>([], set => {
+  window.getKubernetesPortForwards().then(value => {
+    set(value);
+  });
+  window.events?.receive('kubernetes-port-forwards-update', (value: unknown) => {
+    set((value ?? []) as UserForwardConfig[]);
+  });
+  return () => {};
```

Comment:
question: do we need this line? (again, we don't do it in the other case)

Reply:
Yes, removing !